armi/reactor/reactorParameters.py

```diff
@@ -198,13 +203,15 @@ def defineCoreParameters():
 
         pb.defParam(
             "referenceBlockAxialMesh",
-            units="cm",
+            units=units.CM,
             description="The axial block boundaries that assemblies should conform to in a uniform mesh case.",
             default=None,
         )
 
         pb.defParam(
-            "critSearchSlope", units=None, description="Critical keff search slope"
+            "critSearchSlope",
+            units=units.UNITLESS,
```

Comment:
Should this be something like `1/cm`, or did you check that it is per insertion-fraction?

Reply:
This "slope" is the result of this function:

https://github.com/terrapower/armi/blob/9c690f0ee6642c8dc3e16166daa2e5db59b9874a/armi/utils/mathematics.py#L520

I was taking a stab based on the docstring:

https://github.com/terrapower/armi/blob/9c690f0ee6642c8dc3e16166daa2e5db59b9874a/armi/utils/mathematics.py#L549-L550

So, the slow of a curve where the vertical axis has units of "number of new fissions per last fission", has units of what?

Reply:
I would call say keff is unitless, it's effectively a ratio. So the slope would be 1/time. I think the time is probably in units of days, so `1/day`?

Reply:
Yeah, I suppose my original, off-the-cuff, was that the since the vertical axis didn't have units, neither did the slope.

But I think it's 1/time, for sure.  And looking at the downstream use of this variable, days is correct.

Nice call!